kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/UploadSourceTask.kt

```diff
@@ -0,0 +1,110 @@
+package org.radarbase.connect.upload
+
+import org.apache.kafka.connect.errors.ConnectException
+import org.apache.kafka.connect.source.SourceRecord
+import org.apache.kafka.connect.source.SourceTask
+import org.radarbase.connect.upload.converter.Converter
+import org.radarbase.connect.upload.api.PollDTO
+import org.radarbase.connect.upload.api.RecordMetadataDTO
+import org.radarbase.connect.upload.api.UploadBackendClient
+import org.radarbase.connect.upload.converter.Converter.Companion.END_OF_RECORD_KEY
+import org.radarbase.connect.upload.converter.Converter.Companion.RECORD_ID_KEY
+import org.radarbase.connect.upload.converter.Converter.Companion.REVISION_KEY
+import org.radarbase.connect.upload.converter.altoida.AltoidaCsvExportConverter
+import org.radarbase.connect.upload.util.VersionUtil
+import org.slf4j.LoggerFactory
+import java.lang.Exception
+
+class UploadSourceTask: SourceTask() {
+    private lateinit var uploadClient: UploadBackendClient
+    private lateinit var converters: List<Converter>
+
+    override fun start(props: Map<String, String>?) {
+
+        val connectConfig = UploadSourceConnectorConfig(props!!)
+        uploadClient = UploadBackendClient(
+                connectConfig.getAuthorizer(),
+                connectConfig.getHttpClient(),
+                connectConfig.getBaseUrl())
+
+        // init default converters
+        converters = mutableListOf(AltoidaCsvExportConverter())
+
+        // init additional converters if configured
+        val additionalConverters = connectConfig.getConverterClasses().map {
```

Comment:
If we are configuring this anyway via classpath, you can leave out the explicit converter list and make the default converter classes list include the `AltoidaConverter`.